fontforgeexe/charview.c

```diff
@@ -12848,7 +12848,7 @@ CharView *CharViewCreateExtended(SplineChar *sc, FontView *fv,int enc, int show
     wattrs.mask = wam_events|wam_cursor|wam_utf8_wtitle|wam_utf8_ititle;
     wattrs.event_masks = -1;
     wattrs.cursor = ct_mypointer;
-    wattrs.utf8_icon_title = CVMakeTitles(cv,buf,sizeof(buf));
+    wattrs.utf8_icon_title = (const char*)CVMakeTitles(cv,buf,sizeof(buf));
```

Comment:
`utf8_icon_title` is marked as `const`. Probably to indicate that `GDrawCreateTopWindow` won't modify it (and it doesn't).